src/ucp/core/ucp_mm.c

```diff
@@ -173,6 +181,11 @@ ucs_status_t ucp_mem_map(ucp_context_h context, void **address_p, size_t length,
     ucp_mem_h memh;
 
     if (length == 0) {
+        if (flags & UCP_MEM_FLAG_ZERO_REG) {
```

Comment:
Don't think so - AFAIK the initial request was to support length==0 (w/o any special flag)